input/pagecontent/reference.md

```diff
@@ -2687,6 +2687,10 @@ To insert a parameterized rule set, use the rule set name with a list of one or
 
 As indicated, the list of values is enclosed with parentheses `()` and separated by commas `,`. If you need to put literal `)` or `,` characters inside values, escape them with a backslash: `\)` and `\,`, respectively. White space separating values is optional, and removed before the value is applied to the rule set definition.
 
+{%include tu-div.html%}
+Alternatively, a parameter value may be surrounded by double square brackets `[[` `]]`. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash. The only exception to this is when the author wants to include `]],` or `]])` as part of the parameter value. In this case, the `)` or `,` following the `]]` must be escaped with a backslash. For example, to include `]],` as part of a parameter value within double square brackets, use `]]\,`. This needs to be done even if there is whitespace between the `]]` and `,` or `)`. For example, to include `]] )` as part of a parameter within double square brackets, use `]] \)`. Additionally, note that the full value must be surrounded with double square brackets in order for this type of processing to occur.
```

Comment:
I would like you to split this into two pieces:

Alternatively, a parameter value may be surrounded by double square brackets `[[` `]]`. Within the double square brackets, literal `)` and `,` characters  do not need to be escaped with a backslash*. Use of double brackets makes sense when a parameter requires multiple escape characters.

*The only exception to this is when the author wants to include `]],` or `]])` as part of the parameter value. In this case, the `)` or `,` following the `]]` must be escaped with a backslash. For example, to include `]],` as part of a parameter value within double square brackets, use `]]\,`. This needs to be done even if there is whitespace between the `]]` and `,` or `)`. For example, to include `]] )` as part of a parameter within double square brackets, use `]] \)`. Additionally, note that the full value must be surrounded with double square brackets in order for this type of processing to occur.